### DISCUSSION TITLE
v2.1.24

### DISCUSSION THREAD

**Discussion by Victor-IX** [Victor-IX/Blender-Launcher-V2#93] [Announcements]
# What's Changed

## 💡 Features:
- Clicking on the Install Button in the download tab now navigates you to the Build in the Library.
- Improved Settings readability.
- New option to Disable automatic build Checking on Startup `(Settings > Blender Builds)`.
- New option to only scrape Daily or Stable builds `(Settings > Blender Builds)`.
- New settings to use Pre-release Updates for the Blender Launcher `(Settings > General)`.
- New settings to use System Title Bar `(Settings > Appearance)`.
- You can now open the last closed Blender file by **shift-clicking** on the **Launch** button.
- Every build is now editable.

## 🛠️ Fixes:
- **Fixed Stable build fetching.**
- Fixed installed versions being flagged as "installed" after uninstalling in the download tab.
- Fixed a bug preventing users from launching **Blender 2.79** and older from the Launcher.
- Fixed the inability to move BL using internal moving panels on some Linux Distributions.

## ⚙️ Others:
- New icon in the Context Menu.
- Implemented Semver to manage the version of the Blender Launcher and Blender builds. (We used non-standard version numbers for this version and the previous one to prevent breaking the updater; we will return to a more standard versioning format in the next update).
- Blender Launcher Settings, Log, and Cache files are now located in the user folder.
    - Windows: `%AppData%\..\Local\Blender Launcher`.
    - Linux Config: `$XDG_CONFIG_HOME` or `~/.config/Blender Launcher`
    - Linux Cache: `$XDG_CACHE_HOME` or `~/.cache`
- Stable Build caching to reduce request amount and speed up the fetching of all the Blender builds.
- Improved Error feedback.
- Shift-clicking `Check` will now refresh Stable and Daily.
- Using argparse for parsing commandline arguments.

<hr /><em>This discussion was created from the release <a href='https://github.com/Victor-IX/Blender-Launcher-V2/releases/tag/v2.1.24'>v2.1.24</a>.</em>

**Comment by eisenbruch** [comment]
Im no longer getting any new builds to download on v2.1.24.

Is this happening for anyone else?

I have not downloaded these builds, and they are not showing up in Blender Launcher:
![image](https://github.com/Victor-IX/Blender-Launcher-V2/assets/6180294/f3b022f1-36b9-47bc-a824-e500dbc52686)

**Reply by Victor-IX** [reply]
Can you send us your log file?
On Windows, it's in `%LOCALAPPDATA%/Blender Launcher` 
on Linux it's in `$XDG_CACHE_HOME` or `~/.cache/Blender Launcher`

**Reply by eisenbruch** [reply]
Yes, here you go. Thanks for taking a look.
[Blender Launcher.log](https://github.com/Victor-IX/Blender-Launcher-V2/files/14899349/Blender.Launcher.log)

**Reply by Victor-IX** [reply]
what value do you have in the Blender Launcher.ini file next to the .log file for the `minimum_blender_stable_version`?

**Reply by eisenbruch** [reply]
minimum_blender_stable_version=4

**Reply by zeptofine** [reply]
Try changing that to 4.0

**Reply by zeptofine** [reply]
Addendum, this was addressed here https://github.com/Victor-IX/Blender-Launcher-V2/commit/836e0d7aab95d1bda6baa0d556e94a316b7586e3 and i forgot to add safeguards if that did occur

**Reply by eisenbruch** [reply]
yep, working now. thanks

**Comment by SmoothTurtle872** [comment]
*Finally* Its fixed

**Comment by wallyma** [comment]
Oh no. trying to run a batch command of smoke Simulation and just moving my files and everything into the root folder To have the executable recognizable from the command line. Well I went ahead and deleted and updated all My versions and of course everything is gone. there's nothing in the recycle bin. is there a way you can just give people the option of "delete to recycle bin" instead of completely wiping it off your hard drive " NO UNDELETE". I work so hard on that some simulation :(

**Reply by wallyma** [reply]
I did a recover file, simulation not there but the file is. All my smoke settings too. so okay...whew.

**Reply by Victor-IX** [reply]
If I understand correctly, you had placed some smoke simulation assets in a Blender version root folder in the Blender launcher library, and when deleting the old version, you lost those files?
